commands/version/package.json

```diff
@@ -49,6 +49,7 @@
     "@lerna/validation-error": "file:../../core/validation-error",
     "chalk": "^2.3.1",
     "dedent": "^0.7.0",
+    "globby": "^9.2.0",
```

Comment:
I would suggest `fast-glob` over all alternatives.

Reply:
Used `globby` because it is used in the other core packages, too. :-)